packages/nimble-blazor/Examples/Demo.Shared/Pages/ComponentsDemo.razor.css

```diff
@@ -29,6 +29,11 @@ p, a {
     margin: var(--ni-nimble-standard-padding);
 }
 
+.table-container { 
+    max-height: 500px;
+    overflow-y: scroll;
```

Comment:
Another nitpick with the scrolling behavior is that this causes the label that says "Table" and the column headers to be scrolled out of view too. Ideally we'd instead set a max height on the nimble table and rely on its scrollbar to scroll just the rendered data. If that's not supported yet then I'm ok with this as a temporary approach but agree that Molly's suggestion is better if it works.

Reply:
Nope. Fixed.

Reply:
@atmgrifter00 I'm guessing you resolved this thread before my comment came through. Reopening to see your thoughts on relying on the table's scroll behavior rather than adding overflow styling to the demo app.

Reply:
So, the behavior of scrolling without the header also scrolling isn't supported yet. I agree that once it is we should change things so that is the behavior we see in the example as well.

Reply:
Sounds good. If there's anything we can do to remind ourselves to clean this code up when we get there, let's do it. (e.g. comments/TODOs linking this code and #865)

Reply:
I've created [this issue](https://github.com/ni/nimble/issues/935) and left a comment to direct the implementer to fix the Blazor example.